src/metadataGeneration/tsoa.ts

```diff
@@ -96,4 +96,6 @@ export namespace Tsoa {
   export interface ReferenceTypeMap {
     [refName: string]: Tsoa.ReferenceType;
   }
+
+  export type GenericTypeMap = Map<string, Map<string, string>>
```

Comment:
Could it be more readable if we introduced type alias like `BaseClassName` or `ResolvedModel`?

Reply:
`ResolvedModel` fits much better, we don't want to imply at any point that a certain class in the chain of extensions is the "base".